README.md

```diff
@@ -28,29 +28,308 @@ Ships with the following computed property macros: `duration`, `humanize`, `loca
+```
+
+| Parameters | Values |
+| ---------- | ------ |
+| `<date>` | Any value(s) [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...) |
+| `<dateA>` | A boundary value(s) [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...) |
+| `<dateB>` | An optional boundary value(s) [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...). If not given `<date>` is assigned now, `<dateA>` is assigned `<date>` and `<dateB>` is assigned `<dateA>`. |
+| `precision` | An optional `String` unit of comparison [precision](https://momentjs.com/docs/#/query/is-between), defaults to `'milliseconds'` |
+| `inclusivity` | An optional `String` indicating inclusivity of the boundaries, defaults to `()`|
+
+Returns a `Boolean` that indicates if `<date>` is between `<dateA>` and `<dateB>` to the `precision` level and with boundary inclusions specified by `inclusivity`. See [`momentjs#is-between`](https://momentjs.com/docs/#/query/is-between).
+
+**Examples**
+
+```hbs
+{{is-between '1995-12-25' '2995-12-25'}} {{!-- true --}}
+{{is-between '1995-12-25' '1995-12-25' '2995-12-25' precision='years' inclusivity='[)'}} {{!-- true --}}
+```
+
+### now
+
+```hbs
+{{now}}
+```
+
+Returns the present Moment.
+
+**Examples**
+
+```hbs
+{{!-- date at time of writing }}
+{{now}} {{!-- Sat Jan 27 2018 11:59:31 GMT-0500 --}}
+{{!-- interval is a common named parameter (see the corresponding section) }}
+{{now interval=1000}} {{!-- <current date and updating every 1-second (1000 milliseconds).> --}}
+```
+
+### unix
+
+```hbs
+{{unix <timestamp>}}
+```
+
+| Parameters | Values |
+| ---------- | ------ |
+| `<timestamp>` | An integer `Number` or `String` value representing the number of seconds since the Unix Epoch (January 1 1970 12AM UTC)|
+
+Returns a Moment corresponding to the `<timestamp>`.
+
+
+**Examples**
+
+```hbs
+{{unix '1516586508'}} {{!-- Sun Jan 21 2018 21:01:48 GMT-0500 --}}
+{{!-- Warning: Passing a literal integer value does not work --}}
+{{unix 1516586508}} {{!-- Invalid date --}}
+```
+
+### Common optional named arguments
+
+All helpers accept the following optional named arguments (even though they are not always applicable):
+
+| Parameters | Values |
+| ---------- | ------ |
+| `locale` | An optional `String` [locale](https://momentjs.com/docs/#/i18n/), to override the default global `moment.locale` |
+| `timeZone` | An optional `String` [time zone](https://momentjs.com/timezone/docs/), defaults to `moment.timeZone` (the default time zone) |
+| `interval` | An optional interval `Number` of milliseconds when the helper should be recomputed |
+| `allow-empty` | An optional `Boolean` to ignore the `Invalid date` output when knowingly passing `null`, `undefined`, or `''`, defaults to `false` |
```

Comment:
This is only true for `null` and `undefined`.  Again, arguably could be made to handle all five cases but that's not the implementation today.